docs/source/tutorial_ph.md

```diff
@@ -0,0 +1,246 @@
+applications, {math}`E` and/or {math}`Q` often are identity matrices.
+
+It is known that if the LTI system is minimal and stable, the following are equivalent:
+
+- The system is passive.
+- The system is port-Hamiltonian.
+- The system is positive real.
+
+See for example {cite}`BU22` for more details.
+
+In pyMOR, there exists a {{ PHLTIModel }} class. As of now, pyMOR only supports
+port-Hamiltonian systems with nonsingular E. {{ PHLTIModel }} inherits from
+{{ LTIModel }}, so {{ PHLTIModel }} can be used with all reductors which expect
+a {{ LTIModel }}. For model reduction, it is often desirable to preserve the
+port-Hamiltonian structure, i.e., to compute a ROM which is also port-Hamiltonian.
+
+A passive {{ LTIModel }} can be converted into a {{ PHLTIModel }} using
+the {func}`~pymor.models.iosys.PHLTIModel.from_passive_LTIModel` function if desired.
+Consequentely, one option to preserve port-Hamiltonian structure is to use a reductor
+which preserves passivity (but returns a ROM of type {{ LTIModel }}), and convert the
+ROM into a {{ PHLTIModel }} in a post-processing step.
+
+## A toy problem: Mass-spring-damper chain
+
+As a toy problem, we use a mass-spring-damper chain, which can be formulated
+as a port-Hamiltonian system (see {cite}`GPBV12`):
+
+```{image} msd_example.svg
+:alt: MSD example
+:width: 60%
+```
+
+Here, the spring constants are denoted by {math}`k_i` and the damping constants by
+{math}`c_i`, {math}`i=1,\dots,n/2`.
+The inputs {math}`u_1` and {math}`u_2` are the external forces on the first two
+masses {math}`m_1` and {math}`m_2`. The system outputs {math}`y_1` and {math}`y_2`
+correspond to the velocities of the first two masses {math}`m_1` and {math}`m_2`.
+The toy problem is included in pyMOR in the {mod}`pymor.models.examples` module as
+{func}`~pymor.models.examples.msd_example`.
+
+## Structure-preserving model order reduction
+
+pyMOR provides three reductors which can be used for model order reduction 
+while preserving the port-Hamiltonian structure:
+
+- pHIRKA ({class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor`),
+- PRBT ({class}`~pymor.reductors.bt.PRBTReductor`),
+- passivity preserving model reduction via spectral factorization
+  ({class}`~pymor.reductors.spectral_factor.SpectralFactorReductor`).
+
+In this section, we apply all three reductors on our toy example and compare
+their performance. All three reductors are described in {cite}`BU22` in more detail.
+
+Note: Currently, the {class}`~pymor.reductors.bt.PRBTReductor` and
+{class}`~pymor.reductors.spectral_factor.SpectralFactorReductor` reductors require
+{math}`D` to be nonsingular. The MSD example has a zero {math}`D` matrix. Therefore,
+we have to add a small regularization feedthrough term. This is a limitation of the
+current implementation, since in the background the so-called KYP-LMI is converted
+into a so-called Riccati equation, which is only possible for nonsingular {math}`D`.
```

Comment:
```suggestion
current implementation, since in the background the KYP-LMI is converted
into a Riccati equation, which is only possible for nonsingular {math}`D`.
```